comfyui_custom_scripts/extensions/webuiNodes.js

```diff
@@ -30,6 +26,16 @@ const ext = {
             },
         };
     },
+    async beforeRegisterNodeDef(node, nodeData) {
+        const iframeInfo = await iframeRegisteredEvent;
+
+        if (!Object.keys(iframeInfo.webuiIoNodeDisplayNameMappings).includes(nodeData.name)) {
+            return;
+        }
+
+        nodeData.display_name = `${nodeData.display_name} - ${iframeInfo.workflowTypeDisplayName}`;
```

Comment:
why is there a `iframeInfo.webuiIoNodeDisplayNameMappings` attribute if we don't use the map and directly use `iframeInfo.workflowTypeDisplayName` instead? 

I get that `webuiIoNodeDisplayNameMappings` is usefull for filtering the nodes. My understanding is that this is a python dict that's translated into a js object. If I change the python dict, then these changes will be reflected in the object, but we are still going to use `workflowTypeDisplayName`, and so the changes won't have any effect. 

This can create problems because it *appears* like the dict in python is a map, but we only use the keys. 

All of this is assuming that the value of the dict is stored somewhere else. This is assumed because the expected behavior here would be to use `iframeInfo.webuiIoNodeDisplayNameMappings[nodeData.name]`, but it wasn't used, and so this means that the dict values are not mapped and are distinct from the possible `iframeInfo.workflowTypeDisplayName`. 

Sorry if this is long, I wanted to make sure to make my point clear! Let me know if I'm misunderstanding anything, or if we should send the values separately from the dict even though we can directly map them with the dict itself.

Reply:
Mhh let me rephrase. My problem is that there are 2 possible ways to get the right name for the node:
1)
```js
        nodeData.display_name = `${nodeData.display_name} - ${iframeInfo.workflowTypeDisplayName}`;
```
2)
```js
        nodeData.display_name = `${nodeData.display_name} - ${iframeInfo.webuiIoNodeDisplayNameMappings[nodeData.name]}`;
```

why not use the 2nd one, and NOT pass the value? If we know the value, why not compare it directly with `nodeData.name` instead of cheking if it's in the map?